tests/suites/test_suite_psa_crypto_slot_management.data

```diff
@@ -225,11 +225,8 @@ invalid_handle:INVALID_HANDLE_CLOSED:PSA_ERROR_INVALID_HANDLE
 invalid handle: huge
 invalid_handle:INVALID_HANDLE_HUGE:PSA_ERROR_INVALID_HANDLE
 
-Key slot count: less than maximum
-many_transient_keys:MBEDTLS_PSA_KEY_SLOT_COUNT - 1
```

Comment:
Why remove this testcase?

Reply:
`many_transient_keys:MBEDTLS_PSA_KEY_SLOT_COUNT - 1` fills all the slots except one, and stops here. ``many_transient_keys:MBEDTLS_PSA_KEY_SLOT_COUNT` goes through the same state, and then additionally checks that it can add one more key. So `many_transient_keys:MBEDTLS_PSA_KEY_SLOT_COUNT - 1` does not add any test coverage.

That's only when `MBEDTLS_TEST_PSA_INTERNAL_KEYS` is 0, though. When it's not, until d66dc64622aaae8c47500ce081f1d6c103e204e0, `many_transient_keys:MBEDTLS_PSA_KEY_SLOT_COUNT` was failing, and `many_transient_keys:MBEDTLS_PSA_KEY_SLOT_COUNT - 1` was doing what the “maximum” test case was trying to do.

After d66dc64622aaae8c47500ce081f1d6c103e204e0, there's just the test coverage I'd intended, which was to fill the key store and then stop. If I hadn't had to investigate and solve the problem of accounting for the internal key, I wouldn't have created the `- 1` test case in the first place.